test/Chirp.Razor.UnitTest/ChirpStorageTests.cs

```diff
@@ -0,0 +1,141 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Xunit;
+using Chirp.Razor.Shared.Storage;
+using Microsoft.Data.Sqlite;
+
+public class ChirpStorageTests
+{
+    private readonly SqliteConnection _connection;
+    private readonly DbContextOptions<ChirpDBContext> _contextOptions;
+
+    public ChirpStorageTests()
+    {
+        _connection = new SqliteConnection("Filename=:memory:");
+        _connection.Open();
+
+        _contextOptions = new DbContextOptionsBuilder<ChirpDBContext>()
+            .UseSqlite(_connection)
+            .Options;
+
+        var context = new ChirpDBContext(_contextOptions);
+        context.Database.EnsureCreated();
+
+        context.AddRange(
+            new Author { AuthorId = 1, Name = "Jens", Email = "test@mail.dk" },
+            new Author { AuthorId = 2, Name = "Børge", Email = "wow@dd.dk" }
+        );
+        context.SaveChanges();
```

Comment:
We should try to move this into a database fixture (so we can use xunits `IClassFixture`) in the future such that we could do integration tests. This is fine for now though.